installation/scripts/prow/ord-test.sh

```diff
@@ -0,0 +1,463 @@
+export MIGRATE_HOME
+export PATH="${MIGRATE_HOME}:${PATH}"
+
+export ARTIFACTS="/var/log/prow_artifacts"
+mkdir -p "${ARTIFACTS}"
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+
+    case ${key} in
+        --*)
+            echo "Unknown flag ${1}"
+            exit 1
+        ;;
+        *)    # unknown option
+            POSITIONAL+=("$1") # save it in an array for later
+            shift # past argument
+        ;;
+    esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+echo "Install java"
+curl -fLSs -o adoptopenjdk11.tgz "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.9.1%2B1/OpenJDK11U-jdk_x64_linux_hotspot_11.0.9.1_1.tar.gz"
+tar --extract --file adoptopenjdk11.tgz --directory "${JAVA_HOME}" --strip-components 1 --no-same-owner
+rm adoptopenjdk11.tgz* 
+
+echo "Install maven"
+curl -fLSs -o apache-maven-3.8.5.tgz "https://dlcdn.apache.org/maven/maven-3/3.8.5/binaries/apache-maven-3.8.5-bin.tar.gz"
+tar --extract --file apache-maven-3.8.5.tgz --directory "${M2_HOME}" --strip-components 1 --no-same-owner
+rm apache-maven-3.8.5.tgz* 
+
+echo "Install migrate"
+curl -fLSs -o migrate.tgz "https://github.com/golang-migrate/migrate/releases/download/v4.15.2/migrate.linux-amd64.tar.gz"
+tar --extract --file migrate.tgz --directory "${MIGRATE_HOME}" --no-same-owner
+rm migrate.tgz* 
+
+echo "-----------------------------------"
+echo "Eenvironment"
+echo "-----------------------------------"
+echo "JAVA_HOME: ${JAVA_HOME}"
+echo "M2_HOME: ${M2_HOME}"
+echo "MIGRATE_HOME: ${MIGRATE_HOME}"
+echo "GOPATH: ${GOPATH}"
+echo "Base folder: ${BASE_DIR}"
+echo "Compass folder: ${COMPASS_DIR}"
+echo "ORD-service folder: ${ORD_SVC_DIR}"
+echo "Artifacts folder: ${ARTIFACTS}"
+echo "Path: ${PATH}"
+echo "-----------------------------------"
+echo "Java version:"
+echo "-----------------------------------"
+java -version
+echo "-----------------------------------"
+echo "Migrate version:"
+echo "-----------------------------------"
+migrate --version
+echo "-----------------------------------"
+echo "Go version:"
+echo "-----------------------------------"
+go version
+echo "-----------------------------------"
+
+echo "Starting compass"
+cd "${COMPASS_DIR}/components/director"
+source run.sh --auto-terminate 180 & 
+
+COMPASS_URL="http://localhost:3000"
+
+START_TIME=$(date +%s)
+until is_ready "${COMPASS_URL}/healthz" ; do
+    CURRENT_TIME=$(date +%s)
+    SECONDS=$((CURRENT_TIME-START_TIME))
+    if (( SECONDS > 300 )); then
+        echo "Timeout of 5 min for starting compass reached. Exiting."
+        exit 1
+    fi
+    echo "Wait 10s before check Director health again ..."
+    sleep 10
+done
+
+. ${COMPASS_DIR}/components/director/hack/jwt_generator.sh
+
+DIRECTOR_TOKEN="$(get_token | tr -d '\n')"
+INTERNAL_TENANT_ID="$(get_internal_tenant | tr -d '\n')"
+echo "Compass is ready"
+
+echo "Starting ord-service"
+cd "${ORD_SVC_DIR}/components/ord-service"
+export SERVER_PORT=8081
```

Comment:
Rename this `SERVER_PORT` to `ORD_SERVER_PORT` so that it's more clear which of the two app (director/ord service) ports we're talking about

Reply:
Not an option, as this is the variable name that is expected from spring-boot.